Bottom: 6c9fcd47e3c084ff437cc57bf9c6bae824f606fc
Top:    e6e6dcf3dd8b39aba6d6a1fde1ad455b3cba177d
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-07 14:01:05 +0800

Refresh of autofs-5.0.7-fix-potential-null-dereference-in-lookup_mount.patch

---

diff --git a/modules/lookup_yp.c b/modules/lookup_yp.c
index a716e1f..72a8baa 100644
--- a/modules/lookup_yp.c
+++ b/modules/lookup_yp.c
@@ -699,7 +699,7 @@ int lookup_mount(struct autofs_point *ap, const char *name, int name_len, void *
 					       mapent, ctxt->parse->context);
 		if (ret) {
 			time_t now = time(NULL);
-			int rv = CHE_OK;
+			int rv = CHE_FAIL;
 
 			cache_writelock(mc);
 			me = cache_lookup_distinct(mc, key);
